test/setup/deploy_agent.sh

```
@@ -0,0 +1,62 @@
+rest_api_schema=http
+if [ "$tls_enable" = "true" ]; then
+  rest_api_schema=https
+fi
+
+# Create a consumer and the consumer id will be used as the consumer name
+if [ ! -f "./test/_output/.consumer_name" ]; then
+  consumer_name=$(curl -s -k -X POST -H "Content-Type: application/json" ${rest_api_schema}://127.0.0.1:30080/api/maestro/v1/consumers -d '{}' | jq '.id')
+  consumer_name=$(echo "$consumer_name" | sed 's/"//g')
+  echo $consumer_name > ./test/_output/.consumer_name
+fi
+export consumer_name=$(cat ./test/_output/.consumer_name)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)